__tests__/app.test.js

```diff
@@ -128,3 +126,51 @@ describe("/api/articles/", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET:200 responds with an array of comments belonging to a single article", () => {
+    return request(app)
+      .get("/api/articles/9/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments.length > 0).toBe(true);
```

Comment:
How many comments are you expecting to be returned? Its important to check the length here to ensure they are all coming back